### DISCUSSION TITLE
Translation of subcategories

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1491] [Q&A]
I would like to have a translation for the names of subcategories for Ukrainian version of the site.
I added the translations in data.yml files in the translation/uk folder but it does not work.

Could you, please, help

**Comment by brockfanning** [comment]
@ukraine-sdg The YAML format is tricky sometimes and you need to be careful with the characters you include. I believe your data.yml file cannot be parsed because it contains some tab characters. You can get clues to these types of problems by digging into the logs of the data build. To do this, here are the steps:

1. Go to the data repository in Github
2. Click on "Actions"
3. Click on the workflow run that you would like to look at (probably the most recent one at the top)
4. Click on "build"
5. Expand (click) the "Build data" section
6. Scroll through and look for clues

In this case, you can see some line numbers with the problem:
* https://github.com/ukraine-sdg/data/runs/4407433738?check_suite_focus=true#step:5:9
* https://github.com/ukraine-sdg/data/runs/4412337338?check_suite_focus=true#step:5:11

Unfortunately it stops parsing as soon as it finds one error, so it does not tell you all of the errors. I believe you have tabs in several lines within those data.yml files. Check lines 59-65 of uk/data.yml, and 50-56 of en/data.yml.

In short, I believe this problem was caused by having tab characters between the ":" and the value, in the YAML files. Make sure to only use a single space between the ":" and the value.

**Comment by LucyGwilliamAdmin** [comment]
@ukraine-sdg did Brock's response resolve your issue?
